app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -16,16 +16,21 @@ function ContainerTopologyCtrl($scope, $http, $interval, topologyService, $windo
   $scope.d3 = d3;
 
   $scope.refresh = function() {
-    var id;
+    var id, type;
     var pathname = $window.location.pathname.replace(/\/$/, '');
     if (pathname.match(/show$/)) {
       id = '';
+    } else if (pathname.match('/(.+)/show/([0-9]+)')) {
+      var arr = pathname.match('/(.+)/show/([0-9]+)')
+      type = arr[1] + '_topology'
+      id = '/' + arr[2]
     } else {
       // search for pattern ^/<controler>/<id>$ in the pathname
+      type = 'container_topology'
```

Comment:
Correct me if I'm wrong please, but .. isn't it *always* `container_topology` here? We don't really reuse the controller for any other topologies now, do we?